[2025-07-31T11:50:12.886+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-31T11:50:12.910+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: world_population_ETL_dag_43.process_data_task scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-07-31T11:50:12.921+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: world_population_ETL_dag_43.process_data_task scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-07-31T11:50:12.921+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-07-31T11:50:12.936+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): process_data_task> on 2025-03-10 00:00:00+00:00
[2025-07-31T11:50:12.966+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=268) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-31T11:50:12.967+0000] {standard_task_runner.py:72} INFO - Started process 285 to run task
[2025-07-31T11:50:12.960+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'world_population_ETL_dag_43', 'process_data_task', 'scheduled__2025-03-10T00:00:00+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/project_01_dag.py', '--cfg-path', '/tmp/tmpxwm7w2lp']
[2025-07-31T11:50:12.970+0000] {standard_task_runner.py:105} INFO - Job 22: Subtask process_data_task
[2025-07-31T11:50:13.060+0000] {task_command.py:467} INFO - Running <TaskInstance: world_population_ETL_dag_43.process_data_task scheduled__2025-03-10T00:00:00+00:00 [running]> on host 4c2d4060a2fe
[2025-07-31T11:50:13.249+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='givi-abe' AIRFLOW_CTX_DAG_ID='world_population_ETL_dag_43' AIRFLOW_CTX_TASK_ID='process_data_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T00:00:00+00:00'
[2025-07-31T11:50:13.250+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-07-31T11:50:14.300+0000] {logging_mixin.py:190} INFO - Connection to the website established.
[2025-07-31T11:50:14.331+0000] {logging_mixin.py:190} INFO - Lists are clear!
[2025-07-31T11:50:14.332+0000] {logging_mixin.py:190} INFO - Lists are full of data!
[2025-07-31T11:50:14.335+0000] {logging_mixin.py:190} INFO - Data processing complete.
[2025-07-31T11:50:14.336+0000] {python.py:240} INFO - Done. Returned value was:                 COUNTRY  ...                    INS_DATE
0                India   ...  2025-07-31 11:50:14.317950
1                China   ...  2025-07-31 11:50:14.318048
2        United States   ...  2025-07-31 11:50:14.318115
3            Indonesia   ...  2025-07-31 11:50:14.318182
4             Pakistan   ...  2025-07-31 11:50:14.318247
..                  ...  ...                         ...
228         Montserrat   ...  2025-07-31 11:50:14.331364
229   Falkland Islands   ...  2025-07-31 11:50:14.331433
230            Tokelau   ...  2025-07-31 11:50:14.331484
231               Niue   ...  2025-07-31 11:50:14.331533
232           Holy See   ...  2025-07-31 11:50:14.331593

[233 rows x 12 columns]
[2025-07-31T11:50:14.405+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-07-31T11:50:14.406+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=world_population_ETL_dag_43, task_id=process_data_task, run_id=scheduled__2025-03-10T00:00:00+00:00, execution_date=20250310T000000, start_date=20250731T115012, end_date=20250731T115014
[2025-07-31T11:50:14.636+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-31T11:50:14.677+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-31T11:50:14.692+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
